rbi/gems/minitest.rbi

```diff
@@ -0,0 +1,5 @@
+# typed: __STDLIB_INTERNAL
+
+class Minitest::Runnable; end
+
+class Minitest::Test < ::Minitest::Runnable; end
```

Comment:
should this really be in Sorbet? If so it should probably be the full minitest RBI

otherwise, if there are individual test cases that need this RBI to be present, this should be included in the relevant test case.

Reply:
Fair point. I was concerned that type checking that was previously passing would suddenly break without the signatures, but I agree this belongs in the apps.